src/main/scala/cromwell/engine/backend/local/SharedFileSystem.scala

```
@@ -130,11 +137,30 @@ trait SharedFileSystem {
       WdlFile(WdlFile.appendPathsWithSlashSeparators("/root", pathWithWorkflowName.subpath(1, pathWithWorkflowName.getNameCount).toString))
     }
 
-    // If this call is using Docker, adjust input paths, otherwise return unaltered input paths.
+    def localizeInputToCallDirectory(path: String): Path = {
+      val originalPath = Paths.get(path)
+      val callPath = Try {
+        val callInputsDirectory = LocalBackend.hostCallPath(workflowDescriptor, call.name, callKey.index).resolve("inputs")
+        if (!callInputsDirectory.toFile.exists) callInputsDirectory.toFile.mkdirs
+        val hash = hashPath(originalPath)
+        callInputsDirectory.resolve(s"$hash-${originalPath.getFileName.toString}")
+      }
+
+      callPath.toOption flatMap { cp =>
+        CallLocalizers map { _(Option(callKey.scope), originalPath, cp) } find { _.isSuccess } map { _ => cp }
+      } getOrElse {
+        logger.warn(s"Unable to isolate call inputs for call ${call.name}")
```

Yep. If we're going this route should we also fail if a task list a `File` output but this file is not present when the task finishes ? (That would be the equivalent of  "delocalizing" a file in JES).